shared/src/metabase/mbql/util.cljc

```diff
@@ -624,6 +624,8 @@
                            max-results)]
     (safe-min mbql-limit constraints-limit)))
 
+;; TODO -- This seems like it would be easily confused with
+;; [[metabase.driver.sql.query-processor/source-query-alias]]. Joins are REQUIRED to have aliases anyway
```

Comment:
Add to known refactorings Notion page?

Reply:
yeah @howonlee already mentioned that, I'm going to add them there